Cilsil/Services/DecompilationService.cs

```diff
@@ -61,6 +61,21 @@ public ServiceExecutionResult Execute()
                         }
                         catch
                         {
+                            try
```

Comment:
What is the purpose here? In case we can't analyze netmodule files, we fall back and analyze dlls only?

Reply:
oops resolved.

Reply:
unnecessary code